CMake/HHVMExtensionConfig.cmake

```diff
@@ -951,6 +951,13 @@ function (HHVM_EXTENSION_INTERNAL_HANDLE_LIBRARY_DEPENDENCY extensionID dependen
       HHVM_EXTENSION_INTERNAL_ADD_LINK_LIBRARIES(${MYSQL_CLIENT_LIBS})
     endif()
   elseif (${libraryName} STREQUAL "pgsql")
+    FIND_PACKAGE(PostgreSQL)
+    if (${PostgreSQL_VERSION_STRING} VERSION_LESS "9.1.0")
+      message(STATUS "System PostgreSQL version is too old: Disabling extension.")
+      HHVM_EXTENSION_INTERNAL_SET_FAILED_DEPENDENCY(${extensionID} ${dependencyName})
+      return()
+    endif()
+
```

Comment:
Drop this change entirely for now, there's work that still needs to be done before this can be switched.